### PR TITLE
Enlarge intrepid helm

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Expanded the Intrepid helm room and moved the sensors console, added a window to allow the sensors console operator to see the people buckled and see the helm, added two softsuit dispensers to fill the space a little, made the window doors towards the cannon station a double window and duplicated it to the right of the sensors/helm area for symmetry.

image

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]

File: html/changelogs/fluffyghost.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Expanded the Intrepid helm room and moved the sensors console, added a window to allow the sensors console operator to see the people buckled and see the helm, added two softsuit dispensers to fill the space a little, made the window doors towards the cannon station a double window and duplicated it to the right of the sensors/helm area for symmetry."
```
Comment (SleepyGemmy) [inline html/changelogs/fluffyghost.yml:41]:
```suggestion
  - maptweak: "Expanded the Intrepid helm room and moved the sensors console, added a window to allow the sensors console operator to see the people buckled and see the helm, added two softsuit dispensers to fill the space a little, made the window doors towards the cannon station a double window and duplicated it to the right of the sensors/helm area for symmetry."
```
the changelog entry is very long. consider making it more succinct, if possible.

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
map corruption.
image

this room has vents, but no air alarm.
image

this room has an air alarm, but no vents. the windoors will probably get annoying as well.
the floor decals do not follow praxis.
image

not sure if these are empty suit storages or voidsuits. we already have some in EVA.
image

these radios are the wrong way around.
image

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
map corruption still there. check the mapdiff.

**Review by SleepyGemmy — DISMISSED** [review]
ditto. same as last review.
image

**Pull request merged**